theme.json

```diff
@@ -0,0 +1,413 @@
+      "defaultDuotone": false,
+      "customGradient": false,
+      "customDuotone": false,
+      "palette": [
+        {
+          "slug": "text",
+          "color": "var(--wp--custom--text--secondary)",
+          "name": "text"
+        },
+        {
+          "slug": "heading",
+          "color": "var(--wp--custom--text--primary)",
+          "name": "heading"
+        },
+        {
+          "slug": "background",
+          "color": "var(--wp--custom--background--primary)",
+          "name": "background"
+        }
+      ],
+      "gradients": []
+    },
+    "custom": {
+      "body": {
+        "1": {
+          "font-family": "Aktiv Grotesk VF",
+          "font-weight": 400,
+          "line-height": "140%",
+          "font-size": "16px"
+        },
+        "2": {
+          "font-family": "Aktiv Grotesk VF",
+          "font-weight": 400,
+          "line-height": "140%",
+          "font-size": "18px"
+        }
+      },
+      "heading": {
+        "1": {
+          "font-family": "Aktiv Grotesk VF",
+          "font-weight": 500,
+          "line-height": "clamp(1.1em, calc(1.25em + -0.4vw), 1.2em)",
+          "font-size": "clamp(1.07rem, calc(1.02rem + 0.25vw), 1.28rem)",
+          "letter-spacing": "-0.01em"
+        },
+        "2": {
+          "font-family": "Aktiv Grotesk VF",
+          "font-weight": 500,
+          "line-height": "clamp(1.1em, calc(1.25em + -0.4vw), 1.2em)",
+          "font-size": "clamp(1.14rem, calc(1.04rem + 0.47vw), 1.53rem)",
+          "letter-spacing": "-0.01em"
+        },
+        "3": {
+          "font-family": "Aktiv Grotesk VF",
+          "font-weight": 500,
+          "line-height": "clamp(1.1em, calc(1.25em + -0.4vw), 1.2em)",
+          "font-size": "clamp(1.30rem, calc(1.08rem + 1.09vw), 2.20rem)",
+          "letter-spacing": "-0.01em"
+        },
+        "4": {
+          "font-family": "Aktiv Grotesk VF",
+          "font-weight": 500,
+          "line-height": "clamp(1.1em, calc(1.25em + -0.4vw), 1.2em)",
+          "font-size": "clamp(1.38rem, calc(1.08rem + 1.52vw), 2.64rem)",
+          "letter-spacing": "-0.01em"
+        },
+        "5": {
+          "font-family": "Aktiv Grotesk VF",
+          "font-weight": 500,
+          "line-height": "clamp(1.1em, calc(1.25em + -0.4vw), 1.2em)",
+          "font-size": "clamp(1.57rem, calc(1.04rem + 2.69vw), 3.81rem)",
+          "letter-spacing": "-0.01em"
+        }
+      },
+      "space": {
+        "padding": {
+          "global": "clamp(1.00rem, calc(0.40rem + 3.01vw), 3.50rem)"
+        },
+        "block": {
+          "gap": "clamp(1.50rem, calc(1.38rem + 0.60vw), 2.00rem)"
+        }
+      },
+      "alignment": {
+        "center": "744px",
+        "wide": "1008px"
+      },
+      "background": {
+        "primary": "#f5f2f1"
+      },
+      "text": {
+        "primary": "#13191e",
+        "secondary": "#444140",
+        "helper": "#6a6766",
+        "interactive": "#9a6014"
```

Comment:
This could probably be something like `brand` instead of interactive, or maybe in addition. I stoke interactive from IBM Carbon and I've regretted it, it's pretty specific to their philosophy. This color should be the brand color and it gets used for a lot of stuff beyond interactive elements.

Reply:
I actually realized that I dropped the usage of it and so I would remove this instead of renaming it. Originally, it was used for the label on a button when it was selected. I took that away because changing the background of the button felt like that something we would want in the starter theme, but the label changing could be left up to the designer/dev for the minimal `theme.json`. They could decide if they want that or not, and what it would be if they did since it is brand specific technically.

Lemme know what you think.